src/mono/wasi/build/WasiApp.InTree.props

```diff
@@ -10,4 +10,11 @@
     <TrimMode>full</TrimMode>
     <RunAnalyzers>false</RunAnalyzers>
   </PropertyGroup>
+  <ItemGroup>
+    <_MonoRuntimeComponentDontLink Include="libmono-component-diagnostics_tracing-static.a"/>
```

Comment:
Please open an issue about it

Reply:
We already have it for long time https://github.com/dotnet/runtime/issues/61294